apps/service_define/lib/define_web/gettext.ex

```diff
@@ -0,0 +1,24 @@
+defmodule DefineWeb.Gettext do
```

Comment:
What's the JSR/need for this guy?

Reply:
Thanks for removing the comments.

Reply:
Not 100% sure, but the controller blows up when we delete it ...

Reply:
Welp, JSR enough for me.